src/typings.ts

```diff
@@ -233,8 +233,18 @@ export interface OAuthSharedParams {
    */
   redirectUri?: string;
 }
-
-export interface AuthorizeOptions extends OAuthSharedParams, ConfigsOptions, AuthorizeConfigsOptions {
+export interface AffiliateTrackingParams {
+  /**
+   * An optional affiliate code used for tracking which affiliate referred the user.
+   *
+   * Typically passed as a query parameter (e.g., `?affiliate_code=mtb_banana`) during OAuth login or signup.
```

Comment:
```suggestion
```

this doesn't really matter for the SDK user. it's an implementation detail that we don't need to leak.